lxd/db/migration.go

```diff
@@ -125,6 +125,8 @@ INSERT INTO nodes(id, name, address, schema, api_extensions) VALUES(1, 'none', '
 INSERT INTO projects (name, description) VALUES ('default', 'Default LXD project');
 INSERT INTO projects_config (project_id, key, value) VALUES (1, 'features.images', 'true');
 INSERT INTO projects_config (project_id, key, value) VALUES (1, 'features.profiles', 'true');
+INSERT INTO projects_config (project_id, key, value) VALUES (1, 'limits.containers', '');
+INSERT INTO projects_config (project_id, key, value) VALUES (1, 'limits.virtual-machines', '');
```

Comment:
No need for that, it's perfectly fine for those keys to not be in the database at all.